pkg/controller/pxc/tls.go

```diff
@@ -149,7 +149,14 @@ func (r *ReconcilePerconaXtraDBCluster) createSSLByCertManager(cr *api.PerconaXt
 			SecretName: cr.Spec.PXC.SSLInternalSecretName,
 			CommonName: cr.Name + "-pxc",
 			DNSNames: []string{
+				cr.Name + "-pxc",
 				"*." + cr.Name + "-pxc",
+				cr.Name + "-haproxy-replicas." + cr.Namespace + ".svc.cluster.local",
+				cr.Name + "-haproxy-replicas." + cr.Namespace,
+				cr.Name + "-haproxy-replicas",
+				cr.Name + "-haproxy." + cr.Namespace + ".svc.cluster.local",
+				cr.Name + "-haproxy." + cr.Namespace,
+				cr.Name + "-haproxy",
```

Comment:
should we add `-haproxy-replicas` if the service was disabled or should we add haproxy at all if we have proxysql only 
@delgod  what do you think ? I think no...

Reply:
I think we should add haproxy always, reason: user can change options anytime after